pkgs/applications/audio/zrythm/default.nix

```diff
@@ -197,7 +174,6 @@ stdenv.mkDerivation rec {
     "-Dmanpage=true"
     "-Drtaudio=enabled"
     "-Drtmidi=enabled"
-    "-Dsdl=enabled"
     # "-Duser_manual=true" # needs sphinx-intl
```

Comment:
I thought this comment was interesting, so I decided to look into it.

[`git blame` shows](https://github.com/NixOS/nixpkgs/blame/e1a4c08ae81a1e4c99b7496bd232cc4e47708957/pkgs/applications/audio/zrythm/default.nix#L201) that it's over 3 years old! Since then, `python312Packages.sphinx-intl` has been added to nixpkgs. I don't know how important this is to people, but given that we have this package now I think this _could_ be enabled. Not sure how important this is to people. 

Thoughts?

cc @tshaynik

Reply:
I don't know what the nix packager way to do is, but, for reference, the python requirements to build the user manual are in `requirements.txt`. This is how you can build the user manual in HTML format in English:
```
python3 -m venv venv
source ./venv/bin/activate
python3 -m pip install -r requirements.txt
# setup meson (meson setup builddir ...)
ninja -C builddir html-manual-en
```

Then the built manual will be available in the builddir under doc/user